src/lv_core/lv_refr.c

```diff
@@ -741,6 +741,13 @@ static void lv_refr_vdb_flush(void)
 
     /*Flush the rendered content to the display*/
     lv_disp_t * disp = _lv_refr_get_disp_refreshing();
+#if LV_USE_GPU
+    if (disp->driver.gpu_wait_cb)
```

Comment:
Just to follow the coding style please write it like this
```c
if (disp->driver.gpu_wait_cb) disp->driver.gpu_wait_cb(&disp->driver);
```

Reply:
ok, I'll change it.